README.md

```diff
@@ -13,9 +13,9 @@ This project used gettext for translations. Gettext uses PO files for translatio
 ## Development
 Listudy is developed in Elixir using the phoenix framework. It uses postgresql as database. 
 To start the server:
-  * You need to install Elixir, npm, and postgresql.
+  * You need to [install Elixir](https://elixir-lang.org/install.html), [install npm](https://www.npmjs.com/get-npm), and [PostgreSQL](https://www.enterprisedb.com/downloads/postgres-postgresql-downloads). The DB user and password for Postgres should be 'postgres' (no quotes).
```

Comment:
Please change the postgres link to this: https://www.postgresql.org/download/

Reply:
Done